tias/src/Components/EmployeeList/EmployeeList.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC } from 'react'
+import { EmployeeRow } from './EmployeeRow'
+import { GenerateButton } from './GenerateButton'
+import { AcceptButton } from './AcceptButton'
+
+interface Props {
+	
+}
+
+function getData(): Array<string> {
```

Comment:
Good forward thinking! This will be easy to convert to an API call. Do you want the format as a JSON of a list of names?

Reply:
That would work, since everything else will be JSON